Assets/MixedRealityToolkit.Extensions/SceneTransitionService/SceneTransitionServiceProfile.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.MixedReality.Toolkit.Utilities;
-using UnityEditor;
```

Comment:
Does using unityeditor disable running doc generation for this file?

Reply:
It has the potential to cause a build break by including the `UnityEditor` namespace in a file included at runtime. (at the moment it doesn't since Unity is for some reason including a class in this namespace at runtime. It used to be a build break and I'd expect it to be again at some point).

The namespace isn't being used in this file.

Reply:
That works, I just wanted to confirm understanding